canonical-web-and-design/vanilla-framework
# Fix Vanilla import causing build failures

## Done

Fixes broken Vanilla imports (when nothing is included).

Fixes #3499

## QA

- Verify that demo works fine without regressions [demo](insert-demo-url)
- All tests should pass
- Check if new "Check if import test file exists but is empty" test step passes:
  - https://github.com/canonical-web-and-design/vanilla-framework/pull/3501/checks?check_run_id=1734569376
